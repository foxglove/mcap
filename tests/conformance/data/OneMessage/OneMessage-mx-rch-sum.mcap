version https://git-lfs.github.com/spec/v1
oid sha256:ddafe120edbb323bb6e3b82995445758799c2074700b39e946f60df623e46a84
size 263
