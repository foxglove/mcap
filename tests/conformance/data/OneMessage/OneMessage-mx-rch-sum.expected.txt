Header library="" profile=""
ChannelInfo id=1 message_encoding=a metadata={foo=bar} schema=b schema_encoding=c schema_name=Example topic=example
Message channel_id=1 log_time=2 message_data=<010203> publish_time=1 sequence=10
ChannelInfo id=1 message_encoding=a metadata={foo=bar} schema=b schema_encoding=c schema_name=Example topic=example
SummaryOffset group_length=66 group_opcode=12 group_start=138
SummaryOffset group_length=0 group_opcode=12 group_start=204
SummaryOffset group_length=0 group_opcode=9 group_start=204
SummaryOffset group_length=0 group_opcode=7 group_start=204
Footer summary_crc=0 summary_offset_start=204 summary_start=138
